tests/Support/GeneratorMockTrait.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace RonasIT\Support\Tests\Support;
+
+use Illuminate\Support\Facades\View;
+use phpmock\Mock;
+use phpmock\MockBuilder;
+
+trait GeneratorMockTrait
+{
+    public function mockNativeFunction(string $namespace, string $name, $result): Mock
```

Comment:
please check that such logic already exists in the laravel helpers package and use it